### PR TITLE
Fix actionpack typos [ci skip]

### PR DESCRIPTION
**Pull request by tomkadwill**
### Summary

Fix typos in actionpack docs.

### CODE REVIEW

File: actionpack/lib/action_controller/metal/request_forgery_protection.rb [hunk 1]
```diff
@@ -199,8 +199,8 @@ def handle_unverified_request
 
     protected
       # The actual before_action that is used to verify the CSRF token.
-      # Don't override this directly. Provide your own forgery protection
-      # strategy instead. If you override, you'll disable same-origin
+      # Don't override this directly, provide your own implementation of `forgery_protection_strategy`
+      # instead. If you override, you'll disable same-origin
```
Comment (vipulnsward) [inline actionpack/lib/action_controller/metal/request_forgery_protection.rb]: Lets continue having this as is.

File: actionpack/lib/action_controller/metal/streaming.rb [hunk 2]
```diff
@@ -73,7 +73,7 @@ module ActionController #:nodoc:
   # or not, is by using +content_for+, +provide+ and +yield+.
   #
   # Take a simple example where the layout expects the template to tell
-  # which title to use:
+  # it which title to use:
```
Comment (vipulnsward) [inline actionpack/lib/action_controller/metal/streaming.rb]: Lets keep this as is.

**Comment by vipulnsward** [comment]
@tomkadwill can you revert the two changes, rest look good.

File: actionpack/lib/action_controller/metal/streaming.rb (hunk 2 above)
Comment (ptrippett) [inline actionpack/lib/action_controller/metal/streaming.rb]: "Take a simple example where the layout expects the template to be told which title to use:" ?

Comment (vipulnsward) [inline actionpack/lib/action_controller/metal/streaming.rb]: No, this is correct as is, "the template tells the layout", not other way around.

**Comment by tomkadwill** [comment]
Thanks @vipulnsward, I've updated the PR with your suggestions.

**Comment by vipulnsward** [comment]
Thanks @tomkadwill !

**Pull request merged**